etc/kayobe/environments/ci-aio/automated-setup.sh

```diff
@@ -6,6 +6,11 @@ cat << EOF | sudo tee -a /etc/hosts
 10.205.3.187 pulp-server pulp-server.internal.sms-cloud
 EOF
 
+if [ "${1-0}" != "--non-lvm" ]; then
```

Comment:
It would be simpler for the user to just test for the existence of any physical volumes or volume groups